packages/sdk-react/src/hooks/useWallet.tsx

```diff
@@ -0,0 +1,28 @@
+import { useQuery } from '@tanstack/react-query';
```

Comment:
do we use this hook?

Reply:
we do in `useFuelAccountConnection`, but anyway if didn't use we should keep it as it's useful for out hooks set